packages/mobile/src/notifications/NotificationList.tsx

```diff
@@ -0,0 +1,95 @@
+import colors from '@celo/react-components/styles/colors'
+import fontStyles from '@celo/react-components/styles/fonts'
+import variables from '@celo/react-components/styles/variables'
+import React, { useEffect } from 'react'
+import { ScrollView, StyleSheet, Text, View } from 'react-native'
+import SafeAreaView from 'react-native-safe-area-view'
+import {
+  NavigationParams,
+  NavigationProp,
+  NavigationScreenProp,
+  NavigationState,
+} from 'react-navigation'
+import { CURRENCIES, CURRENCY_ENUM } from 'src/geth/consts'
+import i18n from 'src/i18n'
+import { headerWithBackButton } from 'src/navigator/Headers'
+import DisconnectBanner from 'src/shared/DisconnectBanner'
+import { getCentAwareMoneyDisplay } from 'src/utils/formatting'
+
+const { contentPadding } = variables
+
+interface OwnProps<T> {
+  dollarBalance: string | null
+  listItemRenderer: (item: T, key: number) => JSX.Element
+  items: T[]
+}
+
+type Props<T> = OwnProps<T>
+
+export function NotificationList<T>(props: Props<T>) {
+  return (
+    <SafeAreaView style={styles.container}>
+      <DisconnectBanner />
+      {props.items.length > 0 ? (
+        <ScrollView>
+          <View style={[styles.scrollArea]}>{props.items.map(props.listItemRenderer)}</View>
```

Comment:
No need for a style array here, there's just one style.
In general, it's not a big deal but I try to avoid style arrays because they cause a component to always be re-rendered when the parent is.